### ISSUE TITLE
size() fails with warning if s-attribute is nested

### ISSUE THREAD

**Issue by ablaette** [PolMine/polmineR#216]
The `size()` method fails with a warning if the structural attribute provided by argument `s-attribute` is a child of the s-attribute the subcorpus has been defined by. See the following example.

```r
library(polmineR)
corpus("GERMAPARL_BETA") %>%
  subset(grep("(16|17|18|19)", lp)) %>%
  size(s_attribute = "party")
```

**Comment by ChristophLeonhardt** [comment]
Solving this should also address #206.

**Comment by ablaette** [comment]
I consider this issue closed - so #206 still needs to be addressed.
